.github/workflows/ci.yaml

```diff
@@ -1,6 +1,8 @@
 # IGNORE ME
 name: CI
 
+import: "goimports"
+
```

Comment:
```suggestion
```

Reply:
You need to setup a new step that install `goimport`

e.g.
```yaml
- name: Install GoImports
  run: go install golang.org/x/tools/cmd/goimports@latest
```

Reply:
Actually need more time to understand the syntax...

Reply:
https://docs.github.com/en/actions/writing-workflows/workflow-syntax-for-github-actions

Reply:
I did add it and it seems correct I think. 
I went back and fort between my original changes, yours and now it seems to finally work.